core/src/main/java/org/everit/json/schema/StringToValueConverter.java

```diff
@@ -142,7 +146,7 @@ private static Number stringToNumber(final String val) throws NumberFormatExcept
             if(bi.bitLength() <= 31){
                 return Integer.valueOf(bi.intValue());
             }
-            if(bi.bitLength() <= 63){
+            if(bi.bitLength() <= MAX_BIT_LENGTH_FOR_LONG){
```

Comment:
Let's be consistent, and either extract both 31 and 63 into static constants, or none of them (see 3 lines above).